### ISSUE TITLE
Problem: Not possible to edit task template after restoring project

### ISSUE THREAD

**Issue by husjon** [ansible-semaphore/semaphore#2022]
### Issue

It seem like it is not possible to edit a task template after restoring a project.
The Edit, Copy/Duplicate and Delete buttons are missing.

This also includes **new** task templates created in the restored project.

### Impact

Web-Frontend (what users interact with), Configuration, Semaphore Project

### Installation method

Docker

### Browser

Firefox, Chrome

### Semaphore Version

```
[docker@nixos:~]$ docker exec -t -i semaphore ash
~ $ semaphore version
v2.9.75
~ $
```

### Logs & errors

No logs and no errors.

Missing buttons
![image](https://github.com/semaphoreui/semaphore/assets/554229/1b8c77fc-a013-42d1-b4c0-d343d40a983b)

Dummy project (manually created) for reference
![image](https://github.com/semaphoreui/semaphore/assets/554229/77bd0292-2c35-4c3e-b18a-fb21742ff095)

### Configuration

```json
{
  "meta": {
    "name": "DiskStation",
    "alert": false,
    "alert_chat": null,
    "max_parallel_tasks": 0
  },
  "templates": [
    {
      "inventory": "NAS",
      "repository": "GitHub",
      "environment": "Empty",
      "name": "Update DiskStation certificate",
      "playbook": "update-diskstation-certificate/playbook.yaml",
      "arguments": "[\"-e\", \"@update-diskstation-certificate/vault.yaml\"]",
      "allow_override_args_in_task": false,
      "description": null,
      "vault_key": "GitHub Vault",
      "type": "",
      "start_version": null,
      "build_template": null,
      "view": null,
      "autorun": false,
      "survey_vars": "null",
      "suppress_success_alerts": false,
      "cron": "0 0 * * 0"
    }
  ],
  "repositories": [
    {
      "name": "GitHub",
      "git_url": "https://github.com/husjon/ansible-semaphore",
      "git_branch": "main",
      "ssh_key": "None"
    }
  ],
  "keys": [
    {
      "name": "None",
      "type": "none"
    },
    {
      "name": "GitHub Vault",
      "type": "login_password"
    }
  ],
  "views": [],
  "inventories": [
    {
      "name": "NAS",
      "inventory": "[nas]\ndiskstation.local",
      "ssh_key": "None",
      "become_key": null,
      "type": "static"
    }
  ],
  "environments": [
    {
      "name": "Empty",
      "password": null,
      "json": "{}",
      "env": null
    }
  ]
}
```

**Comment by husjon** [comment]
Hmm, I'm not entirely sure how this happened, but I decided to spin up a new container and it is now working as expected. I guess it must have been something weird with my original instance, I've now replaced it.

Closing.

**Comment by Yuri-Lima** [comment]
@husjon that is actually an issue. I have opened and they are WIP. #1990

**Comment by husjon** [comment]
@Yuri-Lima ah, nice catch, didn't see that issue when I created mine.
I'll re-open this issue in case the information might help.

**Comment by tobiasschlatter** [comment]
I'm have the same problem. Is there a way to work around this issue until a fix is provided? Thanks!

**Comment by neocharles** [comment]
Are there any updates to this? I am stuck being unable to delete templates on v2.10.35-a71a3c56-1731092948

**Comment by fiftin** [comment]
@neocharles what database? Looks like it is other issue.

**Comment by husjon** [comment]
Hey @fiftin, which release was this fixed in? I couldn't see any mentioned of it in the releases and no issues / PRs mentioning this issue. :)
Thanks

**Comment by neocharles** [comment]
> @neocharles what database? Looks like it is other issue.

Turns out I think I was missing where to actually click.  I didn't realize I had to open the actual template first (by clicking it's name) and then it would show the delete icon.  I think I was assuming I'd see the delete icon on the list of all the tasks, to enable checkboxes behind the templates or something.  (Looking back at it, it's wildly obvious now).
